docs/loaders/README.md

```diff
@@ -0,0 +1,74 @@
+
+# Loaders documentation
+
+Loaders in spetlr:
+
+- [SCD2Loader](#SCD2Loader)
+ 
+
+# SCD2Loader
+
+The SCD2Loader is a class designed for handling Slowly Changing Dimensions (SCD) Type 2 in PySpark dataframes. 
+This loader is particularly useful in scenarios where data evolves over time, and there is a need to maintain 
+historical versions of data records. It enables upserts to SCD2 tables, meaning it can both update existing records and
+insert new records as needed.
+
+The loader adds `ValidFrom`, `ValidTo`, `IsCurrent`, and `HashKey` columns to the source schema. The ValidFrom and ValidTo 
+columns help in tracking the duration for which a record is valid. The IsCurrent flag indicates whether a record is the 
+latest version. The HashKey is a unique identifier for each record, typically generated by hashing one or more columns of the data.
```

Comment:
Recommendation: Add liquid clustering to your source and sink table. Cluster by the join columns and the hashvalue column. Link: https://docs.databricks.com/en/delta/clustering.html